apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -0,0 +1,99 @@
+import { OpenAI } from 'openai';
+
+import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
+
+import { QuestionGenerationContextEmbeddingSchema } from '../../lib/db-types.js';
+import { createServerJob } from '../../lib/server-jobs.js';
+
+import { createEmbedding, openAiUserFromAuthn, vectorToString } from './contextEmbeddings.js';
+
+const sql = loadSqlEquiv(import.meta.url);
+
+/**
+ * Generates the HTML and Python code for a new question using an LLM
+ * @param client the OpenAI client to use
```

Comment:
(applies to all JSDoc comments)

Nits:
* Write the description as a full sentence, ending in a period.
* Leave a blank line after the description.
* Describe each param (and return value) with a full sentence, starting with a capital letter and ending in a period.

Example: https://github.com/PrairieLearn/PrairieLearn/blob/5706f9cd231deb7a30a2bab864ea9513af3aa13b/packages/html/src/index.ts#L98-L103

Reply:
Done in f6f87b58cd4b2e7125697e3a5a4cc77fa2459053.